www/js/components/add-point-card.js

```diff
@@ -96,6 +91,31 @@ export class AddPointCard extends Component {
     this.setState({'description': newText});
   }
 
+  onNameUpdate(newText) {
+    this.setState({'name': newText});
+  }
+
+  onPhotoAdd() {
+    // This logic will not work on the browser
+    // because of issue - Apache Cordova / CB-9852
+    // https://issues.apache.org/jira/browse/CB-9852
+    navigator.camera.getPicture(
+      imgSrc => {
+        this.setState({ imgSrc });
+        BlobUtil.imgSrcToBlob(this.state.imgSrc).then(blob => {
+          this.setState({'imgBlob':blob});
+        })
+      },
+      err => console.error( err ),
+      { sourceType:navigator.camera.PictureSourceType.PHOTOLIBRARY,
```

Comment:
No need to change, but curious can we change the `PictureSourceType` for a "take photo" button?